catalogData/redis32/3x-ha/k8s/statefulset-server.json

```diff
@@ -0,0 +1,153 @@
+      }
+    },
+    "serviceName": "$idx_and_short_serviceid-server",
+    "affinity": {
+      "podAntiAffinity": {
+        "requiredDuringSchedulingIgnoredDuringExecution": [
+          {
+            "topologyKey": "kubernetes.io/hostname",
+            "labelSelector": {
+              "matchLabels": {
+                "idx_and_short_serviceid": "$idx_short_serviceid",
+                "component": "server"
+              }
+            }
+          }
+        ]
+      }
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "component": "server",
+          "service_id": "$service_id",
+          "idx_and_short_serviceid": "$idx_and_short_serviceid",
+          "managed_by": "TAP"
+        }
+      },
+      "spec": {
+        "initContainers": [
+          {
+            "name": "redis-server-init",
+            "image": "18fgsa/redis-server-init:3.2.10",
+            "volumeMounts": [
+              {
+                "name": "opt",
+                "mountPath": "/opt"
+              }
+            ]
+          }
+        ],
+        "containers": [
+          {
+            "args": [
+              "/opt/redis.conf"
+            ],
+            "command": [
+              "/opt/bin/k8s-redis-ha-server"
+            ],
+            "env": [
+              {
+                "name": "SERVICE",
+                "value": "$idx_and_short_serviceid-server"
+              },
+              {
+                "name": "SERVICE_PORT",
+                "value": "redis-server"
+              },
+              {
+                "name": "SENTINEL",
+                "value": "$service_id-sentinel"
+              },
+              {
+                "name": "SENTINEL_PORT",
+                "value": "redis-sentinel"
+              },
+              { "name": "MANAGED_BY", "value":"TAP" },
+              { "name": "REDIS_PASSWORD",
+                "valueFrom": {
+                  "secretKeyRef": {
+                    "name": "$short_serviceid-redis-ha-credentials",
+                    "key": "redis-password"
+                  }
+                }
+              }
+            ],
+            "image": "redis:3.2.10",
+            "imagePullPolicy": "IfNotPresent",
+            "name": "redis-server",
+            "ports": [
+              {
+                "containerPort": 6379,
+                "name": "redis-server"
+              }
+            ],
+            "readinessProbe": {
+              "exec": {
+                "command": [
+                  "redis-cli",
+                  "info",
+                  "server"
+                ]
+              }
+            },
+            "volumeMounts": [
+              {
+                "mountPath": "/data",
+                "name": "data"
+              },
+              {
+                "mountPath": "/opt",
+                "name": "opt"
+              }
+            ]
+          }
+        ],
+        "volumes": [
+          {
+            "emptyDir": {},
```

Comment:
I thought so to keep consistency with the way we have the Sentinels configured & since `/opt` is really only there for the init containers and gets created every time. I see it as a separation of concerns between the directories. The `/opt` for [scripts we manage](https://github.com/18F/kubernetes-release/pull/61#issuecomment-320025579) `/data` for the server's state stuff, e.g. `*.rdb`, `*.aof`, etc.